openpype/hosts/max/plugins/publish/extract_pointcloud.py

```diff
@@ -137,11 +133,11 @@ def get_operators(self, container):
 
         return opt_list
 
-    def get_custom_attr(self, operator):
+    def get_custom_attr(self, point_cloud_settings, operator):
```

Comment:
I'd expect the "settings" to be the last argument because the operator feels more important.  Also it could directly pass the `attribute` dict in the `point_cloud_settings`. Thoughts?